### DISCUSSION TITLE
Please make the comment feature support Japanese.(Safari)

### DISCUSSION THREAD

**Discussion by dninomiya** [vercel/community#4862] [Bug Report]
**⚠️ Japanese users will not be able to comment in Safari until this issue is resolved.**

### Page to Investigate

https://vercel.com/docs/workflow-collaboration/vercel-toolbar

### Steps to Reproduce

In Japanese text input scenarios, pressing the Enter key while typing is common. However, currently pressing Enter triggers the sending process, resulting in texts being sent before the input is complete.

https://github.com/vercel/community/assets/5842851/20af84ab-2f81-4e37-8e15-834c65bb1d35

This issue is occurring in many forms, so we are creating a website to propose solutions. We hope it will be useful for reference in making fixes.👇🏻

https://dninomiya.github.io/form-guide/stop-enter-submit

ChatGPT also had a similar problem, but it has been corrected based on this site.

Thank you for your support!

**Comment by swarnava** [comment]
Hi,

Thanks for feedback! I have forwarded these details to respective team to take further look. Unfortunately we don't have any concrete ETA when we will support this.

Have a great rest of your day!

**Comment by dninomiya** [comment]
@swarnava 

Thank you for your response!

I wanted to emphasize that as long as this issue persists, we are unable to use the commenting feature on Safari. This is becoming a significant obstacle for both clients and developers in implementing the commenting functionality.

We are hopeful for an early resolution to this problem.

Best regards,😌

**Reply by swarnava** [reply]
Hi @dninomiya, This is only known for Safari so far and we fixed it for other browser already. Are you still facing issue on other browser as well?

**Reply by dninomiya** [reply]
@swarnava 
As you pointed out, this issue does not occur on browsers other than Safari. Both I and our clients primarily use Safari, so we continue to hope for a resolution to this problem.

ChatGPT also faced a similar issue, which had been causing problems for Safari users in Japan. However, it has recently been addressed and now functions smoothly.

While it can essentially be considered a browser bug, I am hopeful that it can be resolved with a minor fix.

**Comment by dninomiya** [comment]
@swarnava 
Why was it closed? In any case, I'm looking forward to the day when I can use the comment function in Safari and Japanese. (This is a hotfix for Japanese users)

Thank you! 🙏🏻

**Reply by swarnava** [reply]
Hi there,

Unfortunately we don't have an ETA when this will be fixed. We will update as soon as we have. Thank you!

**Comment by smaeda-ks** [comment]
Thanks for reporting. The fix is in review and should be deployed next week.

**Reply by smaeda-ks** [reply]
This is now fixed

**Comment by dninomiya** [comment]
@smaeda-ks @swarnava 
Thank you for your quick response! 🥳
